ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKSourceFactoryTest.java

```diff
@@ -176,12 +177,10 @@ public void shouldBuildWindowedTable() {
   }
 
   @Test
-  public void shouldBuildNonWindowedTable() {
+  public void shouldBuildV1NonWindowedTable() {
     // Given:
-    when(dataSource.getDataSourceType()).thenReturn(DataSourceType.KTABLE);
-    when(keyFormat.isWindowed()).thenReturn(false);
-    when(keyFormat.getWindowInfo()).thenReturn(Optional.empty());
-    when(keyFormat.getFormatInfo().getFormat()).thenReturn("asdf");
+    setUpForNonWindowedTable();
+    when(ksqlConfig.getBoolean(KsqlConfig.KSQL_ROWPARTITION_ROWOFFSET_ENABLED)).thenReturn(false);
```

Comment:
This mock is not necessary. The default mockito return value for boolean methods is false anyway. We can make the tests more readable by only mocking the necessary methods (i.e., the ones where this config returns true).